src/Options.js

```diff
@@ -144,7 +145,7 @@ addFeatureOption('require', EXPERIMENTAL);
 addFeatureOption('symbols', EXPERIMENTAL);
 addFeatureOption('types', EXPERIMENTAL);
 addFeatureOption('memberVariables', EXPERIMENTAL);
-
+addFeatureOption('forOn', EXPERIMENTAL);
```

Comment:
keep alphabetical order (I know it got out of sync because I am the only one that is obsessive enough about this)